packages/thirdweb/src/chainsaw/paging.ts

```diff
@@ -0,0 +1,14 @@
+import type { ChainsawPagingParams } from "./types.js";
+
```

Comment:
@internal tag

Reply:
Done in [cca8c79](https://github.com/thirdweb-dev/js/pull/4004/commits/cca8c79d6a797aeed7cfe96585488632e1f532d9)